### ISSUE TITLE
Step forward does not work on my system (neither 5sec, 60sec ...)

### ISSUE THREAD

**Issue by Gloster10** [vivictorg/vivictpp#19]
Ubuntu 20.04.03, 5.15.0-43-generic
vivictpp Vers.: Vivict++ 0.1.14 b8f44be

I did compare h.262 videos with h.264 (h.265) videos. Step backward does work, but not step forward (5sec, 60sec, 10min). I did check other key-features, as like Decrease/Increase left frame offset, it does work (to check us-key layout). Step forward/backward does also work, but vice versa: "," => Step backward, "." Step forward.
I did upload the films for testpurposes, please refer to the given link : https://we.tl/t-UiqkwPysw9 (1 week alive).

(An additional hint, a tap on the touchpad start/stop the film.)

If additional information will be needed, I will added, thank you for help.

**Comment by grusell** [comment]
Sorry I have missed this issue. Is this still a problem with the latest version?

If it is, I would ask you to upload the example files again and I will have a look. Also, did you install from snap or build from source?

**Comment by Gloster10** [comment]
I use Vivict++ 0.2.3 cf2a030, build from source, Ubuntu 22.04, Gnome, all snaps removed from the system.
`meson compile -C builddir`
leads to a system crash on my system, but 
`cd builddir`
`sudo ninja`
did work.
I did check it with the uploaded files, same behaviour as by former vivictpp version, please see above.

The mp4 file = h.264, the mkv = h.265, a bit longer then 21 minutes.
You can find the files : https://we.tl/t-CYIdIC7Wgb

Thank you.

**Comment by Gloster10** [comment]
Back to 0.2.0, I did found the slider time bar (only to set the mouse pointer to the lower part of the screen), a very positive surprise.
But it works only until t=18min, then the bar exceeds the screen.
Is there any possibility to denied it by user input. In the help, I did not found anything ?

**Comment by Gloster10** [comment]
I did compile successfully 0.2.3 (main) and now I see a further developed slider time bar. The slider bar does not exceed the screen anymore, and with some carefully clicks, I got the 30min time position with 5 clicks (stop the screen, set the new time position, a.s.o..). If somethimg goes wrong (videos are not synchronized any more, I can click back to a former working position.).
From my side of view a good idea, and well done Grusell.

**Comment by grusell** [comment]
Glad to hear you find the slider time bar useful!

Do you still have problems with seeking with the keyboard? It works find for me with the linked files, once I set frame offset to '2' to compensate for the files having different starting timestamps.

**Comment by Gloster10** [comment]
Yes, I did check it again with the new compiled 0.2.3., but I got the same behaviour as before. 
I have no problems any more with the starting timestamps since I use the script, to set the time stamps of both files to 0, and use the new mp4 container.
"u", "i", ",", ".", "<", ">" does work on my system, and I must confess, with the slider time bar, there is no need to have on my system further keyboard possibilities to step forward (5sec, 60sec, 10min).
Obviously I am the only one with this problem, I will close this issue.
